reporting/src/main/java/org/wso2/testgrid/reporting/TestReportEngine.java

```diff
@@ -76,8 +82,10 @@
     private static final String REPORT_MUSTACHE = "report.mustache";
     private static final String PERFORMANCE_REPORT_MUSTACHE = "performance_report.mustache";
     private static final String EMAIL_REPORT_MUSTACHE = "email_report.mustache";
+    private static final String SUMMARIZED_EMAIL_REPORT_MUSTACHE = "summarized_email_report.mustache";
```

Comment:
We need to merge these two reports together. May be in a later PR.

Reply:
+1